mlrun/common/schemas/function.py

```diff
@@ -140,3 +140,27 @@ class Function(pydantic.v1.BaseModel):
 
     class Config:
         extra = pydantic.v1.Extra.allow
+
+
+class BatchingSpec(pydantic.v1.BaseModel):
```

Comment:
Lastly this can be used only by a specific nuclio version. Should be reflected with a decorator as a feature gate, right?

Reply:
💯